src/helpers/notification_helpers.ts

```diff
@@ -1,45 +1,143 @@
 import { EventFromDevice } from "akeyless-types-commons";
+import { isIccid, isInternationalPhoneNumber, isLongPhoneNumber } from "./phone_number_helepers";
+import { Twilio } from "twilio";
+import { Timestamp } from "firebase-admin/firestore";
+import { v4 as uniqId } from "uuid";
 
-export const send_sms = async (phone_number: string, text: string, entity_for_audit: string): Promise<void> => {
+type SmsFunction = (number: string, text: string, entity_for_audit?: string) => Promise<void>;
+
+const send_local_sms: SmsFunction = async (number, text) => {
+    const {
+        sms_provider: { multisend },
+    } = cache_manager.getObjectData("nx-settings");
+    const msgId = uniqId();
+    let data = new FormData();
+    data.append("user", multisend.user);
+    data.append("password", multisend.password);
+    data.append("from", multisend.from);
+    data.append("recipient", number);
+    data.append("message", text);
+    data.append("customermessageid", msgId);
+
+    const config = {
+        method: "post",
+        maxBodyLength: Infinity,
+        url: "https://api.multisend.co.il/v2/sendsms",
+        data: data,
+    };
+    const response = await axios(config);
+    if (response.status !== 200) {
+        throw `http request to multisend status ${response.status}`;
+    }
+    if (!response.data.success) {
+        throw `http request to multisend error ${JSON.stringify(response.data.error)}`;
+    }
+
+    await add__new_sms_to_db(number, text, "multisend", msgId);
+    logger.log("send_local_sms. message sent successfully", { number, text, response: response.data });
+};
+
+const send_international_sms: SmsFunction = async (number, text) => {
+    const {
+        sms_provider: { twilio },
+    } = cache_manager.getObjectData("nx-settings");
+    const twilioClient = new Twilio(twilio.account_sid, twilio.token);
+    const message = await twilioClient.messages.create({
+        messagingServiceSid: twilio.messaging_service_sid,
+        body: text,
+        to: number,
+    });
+    if (message.errorMessage) {
+        throw `twilioClient.messages.create failed: ${message.errorMessage} `;
+    }
+    await add__new_sms_to_db(number, text, "twilio", message.sid);
+    logger.log("send_international_sms. message sent successfully", { number, text, response: message });
+};
+
+const login_to_monogoto = async () => {
     try {
-        const { sms_provider } = cache_manager.getObjectData("nx-settings");
-        let data = new FormData();
-        data.append("user", sms_provider.user);
-        data.append("password", sms_provider.password);
-        data.append("from", sms_provider.from);
-        data.append("recipient", phone_number);
-        data.append("message", text);
-        let config = {
+        const {
+            sms_provider: { monogoto },
+        } = cache_manager.getObjectData("nx-settings");
+        const data = { UserName: monogoto.UserName, Password: monogoto.Password };
```

Comment:
change Password -> password, UserName -> user (like in other sms settings)

Reply:
done

Reply:
please change user_name -> user